docs/guide/blog/index.md

```diff
@@ -126,9 +143,9 @@ func (k msgServer) CreatePost(goCtx context.Context, msg *types.MsgCreatePost) (
   ctx := sdk.UnwrapSDKContext(goCtx)
   // Create variable of type Post
   var post = types.Post{
-    Creator: msg.Creator,
-    Title:   msg.Title,
-    Body:    msg.Body,
+     Creator: msg.Creator,
```

Comment:
The indentation was correct. We should revert to the original.

Reply:
this isn't an indentation issue, was likely a merge conflict